contents/blog/is-google-analytics-hipaa-compliant.md

```diff
@@ -0,0 +1,68 @@
+
+- **Business Associate Agreement:** An agreement between a covered entity and third party that handles their PHI; it ensures the Business Associate shares the same legal requirements and liability as the Covered Entity   
+
+## Why isn't Google Analytics HIPAA compliant?
+
+Because Google doesn't allow Covered Entities to enter into a BAA with it, as this [disclaimer](https://support.google.com/analytics/answer/6366371#zippy=%2Cin-this-article) on its website explains:
+
+> Unless otherwise specified in writing by Google, Google does not intend uses of Google Analytics to create obligations under the Health Insurance Portability and Accountability Act, as amended, (“HIPAA”), and makes no representations that Google Analytics satisfies HIPAA requirements. If you are (or become) a Covered Entity or Business Associate under HIPAA, you may not use Google Analytics for any purpose or in any manner involving Protected Health Information unless you have received prior written consent to such use from Google.
+
+Because using Google Analytics requires transmitting data to Google-owned servers, a BAA would be necessary to be compliant. Were you to use Google Analytics to process or transmit PHI you would be liable for investigation and a fine.
+
+## How much does violating HIPAA cost?
+
+HIPAA fines operate on a sliding scale based on the severity of the breach and the total number of breaches. This means they get expensive very quickly, especially for products or businesses with large user bases. The largest HIPAA fine to date is [$16 million](https://www.hipaajournal.com/anthem-inc-settles-state-attorneys-general-data-breach-investigations-and-pays-48-2-million-in-penalties/) against health insurer Anthem.
+
+## HIPAA-compliant alternatives to Google Analytics
+
+While Google Analytics is sufficient if you just want to see what pages a user views on a website, it won't help you improve your product in a meaningful way. For that, you need product analytics.
+
+Product analytics platforms like [PostHog](https://posthog.com/) include features like [session recording](https://posthog.com/product/session-recording), [heatmaps](https://posthog.com/product/heatmaps) and [funnels](https://posthog.com/product/funnels) that help you understand how users navigate your product — not just what they're looking at.
+
+If you want to find [product market fit](https://posthog.com/blog/how-to-product-market-fit), your product's [North Star metric](https://posthog.com/blog/north-star-metrics) or what your [users really need](https://posthog.com/blog/how-to-work-out-what-users-need)... you need product analytics.
+
+While there are lots options out there, we think **PostHog is the most HIPAA-compliant product analytics platform** out there.
+
+Unlike third-party analytics options like Mixpanel or Amplitude, **PostHog is the only comprehensive product analytics suite you can self-host** on your own infrastructure. 
```

Comment:
Does this claim sufficiently address the Matomo/Piwik objection? Maybe 'platform' instead of 'suite'

Reply:
Can change, but what was the objection?

Reply:
You could argue that PostHog isn't the only comprehensive product analytics suite you can self-host, as you can self-host Matomo. 

I suppose you could say 'ah yes, but Matomo isn't comprehensive or a suite', but I think saying 'platform' makes us sound broader and more differentiated.